package.json

```diff
@@ -18,7 +18,7 @@
         "@babel/preset-env": "^7.24.5",
         "@chakra-ui/react": "^2.8.2",
         "@deriv-com/analytics": "^1.10.1",
-        "@deriv-com/api-hooks": "^1.4.9",
+        "@deriv-com/api-hooks": "^1.5.3",
```

Comment:
just curious, how is it fixed in api-hooks?

Reply:
in api hooks, there were fixes related to how we handle the reconnection to the websocket in the api hooks. I found that when i remove the provider for ApiProvider from api-hooks, it caused the issue, and yashim had refactored the way we connect to the websocket